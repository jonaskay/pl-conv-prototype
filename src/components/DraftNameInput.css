.DraftNameInput {
  background: transparent;
  border: none;
  border-bottom: 2px solid rgba(255,255,255,.12);
  color: rgba(255,255,255,.9);
  font-size: 1.5rem;
  font-weight: 700;
  margin-top: 1rem;
  outline: none;
  padding: .5rem 0; 
  position: relative;
  width: 100%;
}

.DraftNameInput:focus {
  border-bottom: 2px solid rgba(255,255,255,.5);
  color: #FFF;
}
