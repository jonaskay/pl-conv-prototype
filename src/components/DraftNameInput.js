import React from 'react';
import './DraftNameInput.css';

class DraftNameInput extends React.Component {
  render() {
    return (
      <input type="text" value={this.props.name} className="DraftNameInput" />
    );
  }
}

export default DraftNameInput;
