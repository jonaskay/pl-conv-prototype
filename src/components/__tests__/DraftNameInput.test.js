import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import DraftNameInput from '../DraftNameInput';

it('renders an input field', () => {
  const component = ReactTestRenderer.create(<DraftNameInput name="Foobar" />);
  expect(component).toMatchSnapshot();
});
