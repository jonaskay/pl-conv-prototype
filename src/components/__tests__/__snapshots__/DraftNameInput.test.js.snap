// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders an input field 1`] = `
<input
  className="DraftNameInput"
  type="text"
  value="Foobar"
/>
`;
